src/game.zig

```diff
@@ -326,6 +326,11 @@ pub const collision = struct {
 
 pub const Gamemode = enum(u8) { survival = 0, creative = 1 };
 
+pub const DamageType = enum(u8) {
+	none,
+	fall,
+};
```

Comment:
Since this enum is sent over the network, please explicitly give each tag a number.